### ISSUE TITLE
TypeError: Cannot read property 'split' of null

### ISSUE THREAD

**Issue by dmitriz** [haoliangyu/pg-reactive#7]
Errors when running the given example
```js
const pgrx = require('pg-reactive')

let db = new pgrx('postgres://localhost/merch');

db.query('SELECT * FROM products')
  .map((row) => row.id)
  .subscribe((id) => {
    console.log('ID: ', id);
  });
```

```sh
/Users/dmitrizaitsev/Dropbox/Sandbox/pg-tests/node_modules/pg-reactive/dist/pg-reactive.js:50
      var auth = params.auth.split(':');
                             ^

TypeError: Cannot read property 'split' of null
    at new pgrx (/Users/dmitrizaitsev/Dropbox/Sandbox/pg-tests/node_modules/pg-reactive/dist/pg-reactive.js:50:30)
    at Object.<anonymous> (/Users/dmitrizaitsev/Dropbox/Sandbox/pg-tests/pg-reactive-test.js:3:10)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

**Comment by haoliangyu** [comment]
Thank you for reporting this issue! The error was caused by a bug in the code to parse the database url. It tried to parse the username and password but none of them existed. I have fixed this at #8 and published a new version `v0.3.3`. Please try :-)

**Comment by dmitriz** [comment]
It works perfectly now! 
Thank you for fixing it so fast!
